library/core/src/num/nonzero.rs

```diff
@@ -381,6 +403,10 @@ where
         }
     }
 
+
```

Comment:
Can you remove the added whitespace?
As a rule of thumb, we want to limit the changes in each PR to only add verification code, so that the overall changes we make to the standard library are for verification only.

Reply:
Removed/Updated